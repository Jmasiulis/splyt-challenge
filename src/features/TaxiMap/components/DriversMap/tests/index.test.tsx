/* eslint-disable @typescript-eslint/camelcase */
/* eslint-disable react/jsx-props-no-spreading */
import React from 'react';
import { shallow } from 'enzyme';

import { DriversMap } from '..';

describe('<DriversMap />', () => {
  const defaultProps = {
    google: jest.fn(),
    driverLocations: [
      {
        driver_id: '1',
        location: {
          latitude: 50,
          longitude: -1,
        },
      },
      {
        driver_id: '2',
        location: {
          latitude: 50,
          longitude: -1,
        },
      },
    ],
  };

  it('should render map with markers', () => {
    const wrapper = shallow(<DriversMap {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
