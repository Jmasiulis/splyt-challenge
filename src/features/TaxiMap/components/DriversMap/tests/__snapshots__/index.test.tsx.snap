// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DriversMap /> should render map with markers 1`] = `
<Map
  center={Object {}}
  centerAroundCurrentLocation={false}
  className="map"
  containerStyle={Object {}}
  google={[MockFunction]}
  initialCenter={
    Object {
      "lat": 51.5049375,
      "lng": -0.0964509,
    }
  }
  style={
    Object {
      "height": "50%",
      "margin": "10px",
      "position": "relative",
      "width": "55%",
    }
  }
  visible={true}
  zoom={13}
>
  <Marker
    key="1"
    name="Marker"
    position={
      Object {
        "lat": 50,
        "lng": -1,
      }
    }
  />
  <Marker
    key="2"
    name="Marker"
    position={
      Object {
        "lat": 50,
        "lng": -1,
      }
    }
  />
</Map>
`;
